src/main/java/core/basesyntax/zoo/Animal.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.zoo;
+
+public abstract class Animal implements Feedable {
```

Comment:
I think, this class is redundant as you don't add any functionality here so you can simply implement feedable by Bird, Fish, Tiger, etc.

Reply:
public void feed(Animal animal) {
        System.out.println("Hello, "
                + animal.getClass().getSimpleName()
                + ". It is time for eating.");
    }
I`ve made this method to make all code shorter because it is used in Classes Tiger, Shark etc. and their realizations are shorter